util/github_util.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/thedevsaddam/gojsonq/v2"
+)
+
+func GetLatestTag(githubPAT, githubOrg, repo string) (string, error) {
+	githubAPIURL := fmt.Sprintf("https://%s:@api.github.com/repos/%s/%s", githubPAT, githubOrg, repo)
```

Comment:
Why do we need to include the GitHub Personal Access Token (PAT) to make this API call?

This appears to work without authentication:

```sh
curl https://api.github.com/repos/metal-toolbox/audito-maldito/releases/latest
```